.github/workflows/seqeradocs-changelog.yml

```diff
@@ -0,0 +1,62 @@
+name: Push release change log to Seqera Docs
+
+on:
+    release:
```

Comment:
tab vs spaces?! 😆

Reply:
hah, no tabs here, just 4-space indentation. I'll change it to 2-space indentation to match the other actions.